sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/siprouting/implementation/models/CommunicationError.java

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.phonenumbers.siprouting.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+/** The Communication Services error. */
+@Fluent
+public final class CommunicationError {
```

Comment:
This error and the equivalent public API replacement `SipRoutingError` appear to be very similar to [`ResponseError`](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/src/main/java/com/azure/core/models/ResponseError.java) in `azure-core`. Should these two classes be replaced with that?

Reply:
It could be replaced. 
But since this PR should be mostly just about adding into an existing SDK, I would probably keep it consistent. Like in the case of PhoneNumberError. This can be refactored in future PR.